Code/Engine/Foundation/Platform/Android/Screen_Android.cpp

```diff
@@ -0,0 +1,15 @@
+#include <Foundation/FoundationPCH.h>
+
+#if EZ_ENABLED(EZ_PLATFORM_ANDROID)
```

Comment:
What I don't like about this approach is that this results in many compilation units that are plain empty and serve no function, but will still be compiled. I would prefer a hybrid approach, where we do the preprocessor decision but also have something in cmake to not compile the cpp in the first place.